spec/features/reviews_spec.rb

```diff
@@ -0,0 +1,124 @@
+require "rails_helper"
+require "pry"
+​
+  feature "Create A Review" do
+    scenario "logged in user successfully creates a new review" do
+
+      new_user = User.create(email: "mckelvey.matt@gmail.com", password: "12345678")
+    ​
+      visit user_session_path
+    ​
+      fill_in 'Email', with: 'mckelvey.matt@gmail.com'
+      fill_in 'Password', with: '12345678'
+    ​
+      click_button "Log in"
```

Comment:
The login code that you have here on lines 7-14 looks like it's being repeated a lot. It should probably be pulled out into a `before` block! I.e.:

```
feature "..." do
  before(:each) do 
    # login code goes here
  end

  scenario "..." do
    ...
  end
end